### PR TITLE
[TECH-394] Improve reporting

### PR DESCRIPTION
**Pull request by Jorg88**
branch: feature/TECH-394-improve-reporting
----
📕 [TECH-394](https://vandebron.atlassian.net/browse/TECH-394) Integrate cypress suites into mpyl jorgpost@vandebron.nl 
Make sure that certain cypress suites get triggered for certain projects when specified in the project.yml.

🏗️ Build [16](https://jenkins.k8s-serv-backend.vdbinfra.nl/job/MPyL%20Pipeline%20-%20Test/job/PR-143/16/display/redirect) ✅ Successful, started by _Jorg Post_  
🚀 *[cloudfront-service](https://cloudfront-service-143.test.nl/)*, *enrichChargeSessionsJob*, *job*, *kong-sync*, *[nodeservice](https://nodeservice-143.test.nl/)*, *[sbtservice](https://sbtservice-143.test.nl/)*  

[TECH-394]: https://vandebron.atlassian.net/browse/TECH-394?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/mpyl/utilities/docker/__init__.py [hunk 1]
```diff
@@ -146,3 +144,17 @@ def login(logger: Logger, docker_config: DockerConfig) -> None:
...
+    container = docker.create(image_name)
+    logger.info(f"Created container {container.id}")
+
+    return container
+
+
+def remove_container(logger: Logger, container: Container) -> None:
+    logger.info(f"Removing container {container.id}")
+    docker.remove(container.id)
+    logger.info(f"Removed container {container.id}")
```
Comment (koeves) [inline src/mpyl/utilities/docker/__init__.py:160]: Maybe these can be debug level

**Review by koeves — APPROVED** [review]
Looks good 👍

File: src/mpyl/utilities/docker/__init__.py (hunk 1 above)
Comment (Jorg88) [reply src/mpyl/utilities/docker/__init__.py:160]: I was kinda following the structure of the other functions (`login`, `build`) that also use info logs

Comment (koeves) [reply src/mpyl/utilities/docker/__init__.py:160]: Guess it's fine like this as well.. though now I'd say those would be better on debug level as well

Comment (Jorg88) [reply src/mpyl/utilities/docker/__init__.py:160]: I just checked when & where we use info vs debug and I think `info` here fits into our current overall logging strategy. It's extra information to show/track what is being in the pipeline

**Pull request merged**